CMakeLists.txt

```diff
@@ -85,6 +87,8 @@ install(
 
 install(
   FILES LICENSE
+  ${CMAKE_CURRENT_BINARY_DIR}/clang
+  ${CMAKE_CURRENT_BINARY_DIR}/clang++
```

Comment:
Where this symlinks will point after installation?

Reply:
Good question. The intention is to have them link to the main bin/szcc file, but I am uncertain about how cmake handles the links.

Reply:
I've checked and looks like cmake installs copy of symlink target. At least that what what I've got for `szcc++`:
```
# ls -l /usr/local/bin
total 24
-rwxr-xr-x 1 root root 10174 May 15 23:21 szcc
-rwxr-xr-x 1 root root 10174 May 15 23:21 szcc++
```

Reply:
It actually didn't copy the files, but it made a hardlink, so not at all what was expected. Fixed in new commit.